sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/__init__.py

```diff
@@ -16,16 +16,19 @@
     FeatureFlagConfigurationSetting,
     SecretReferenceConfigurationSetting,
 )
+from ._generated.models import Snapshot, ConfigurationSettingFilter
 from ._version import VERSION
 from ._azure_appconfiguration_error import ResourceReadOnlyError
 
 __version__ = VERSION
 __all__ = [
     "AzureAppConfigurationClient",
     "ConfigurationSetting",
+    "ConfigurationSettingFilter",
```

Comment:
Why we want to expose `ConfigurationSettingFilter`?

Reply:
Yeah - doesn't look like theres a need for this to be public.

Reply:
`Snapshot` has an attribute in type `ConfigurationSettingFilter`. Since we make the generated model `Snapshot` to be public, will also need to make its attributes' non-primary types to be public.